services/metadata/frontend/Header.svelte

```diff
@@ -151,17 +153,17 @@
   .menu-item:hover {
     background-color: var(--cl-transparent-light);
   }
-
-
-
-  @media screen and (min-width: 450px){
+  @media screen and (min-width: 576px) {
 	  /* extra small screen styles */
     .menu {
       width: 20rem;
       height: 100vh;
       position: absolute;
       right: 0px;
     }
+    .title {
+      font-size: 1rem;
+      padding: 12px;
+    }
   }
-
 </style>
```

Comment:
How do you feel now: should this be broken up into multiple components?

Reply:
Not for now, let's keep it like it is for now.

Reply:
ok